modules/node/src/admin/admin.service.ts

```diff
@@ -52,16 +54,25 @@ export class AdminService {
   /**  Get Channels by xpub */
 
   async getChannelStates(userPublicIdentifier: any): Promise<CFCoreRecord[]> {
-    // get channel by xpub
+    // get channel by xpubgetChannelStateByMultiSig
```

Comment:
how is this being passed in? the `userPublicIdentifier` should be typed as a string and used directly, this is a little confusing

Reply:
changed to string.

what do you mean by used directly?